picklists/ThreatIntelFeedType.json

```diff
@@ -14,7 +14,6 @@
             "icon": null,
```

Comment:
there is an item added to Tasks Types picklist also. please include that

Reply:
Included TaskType picklist